libraries/AP_RCProtocol/AP_RCProtocol.cpp

```diff
@@ -243,13 +243,15 @@ void AP_RCProtocol::check_added_uart(void)
             added.uart->set_options(added.uart->get_options() | AP_HAL::UARTDriver::OPTION_RXINV);
             break;
         case CONFIG_420000_8N1:
+#if !APM_BUILD_TYPE(APM_BUILD_iofirmware)
```

Comment:
I think this probably means you need to change the phase check below to eliminate CONFIG_420000_8N1 on IOMCU